pkg/parser/ast/misc.go

```diff
@@ -3930,10 +3930,8 @@ type TableOptimizerHint struct {
 	Indexes []CIStr
 }
 
-// HintTimeRange is the payload of `TIME_RANGE` hint
-type HintTimeRange struct {
-	From string
-	To   string
+type LeadingList struct {
+	Items []interface{}
```

Comment:
Please add some comments about this structure and `Items` like how the corresponding hints might look like.